controllers/homework.js

```diff
@@ -79,6 +79,12 @@ const getCreateHandler = (service) => {
             if ((!req.body.lessonId) || (req.body.lessonId && req.body.lessonId.length <= 2)) {
                 req.body.lessonId = null;
             }
+            if(req.body.dueDate == "__.__.____ __:__"){
```

Comment:
can we make this more generci, for example if parsing the date fails this will be undefined

Reply:
You are right, I've checked again and saw, that we already have such an fallback. But for any reason the fallback is not working on the production system. If I create an task on my local test server with the master branch without any date set everything works fine. If I do the same thing on schul-cloud.org I get the alert "Das Beginndatum muss vor dem Abgabedatum liegen!".
Do you have any idea why?